### ISSUE TITLE
BUG: Wrong result of Kurtosis

### ISSUE THREAD

**Issue by xitongsys** [pandas-dev/pandas#59572]
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the [main branch](https://pandas.pydata.org/docs/dev/getting_started/install.html#installing-the-development-version-of-pandas) of pandas.

### Reproducible Example

```python
a = pd.Series([0,0,0,0,0.00005])

a.kurtosis()
Out[205]: np.float64(0.0)

scipy.stats.kurtosis(a,bias=False)
Out[206]: np.float64(4.999999999999997)
```

### Issue Description

Pandas can't give a right result of kurtosis. But the `scipy.stats.kurtosis` can.

### Expected Behavior

see the code

### Installed Versions

<details>

Replace this line with the output of pd.show_versions()

</details>

**Comment by fbourgey** [comment]
take

**Comment by Liam3851** [comment]
This appears to be a duplicate of #57972. Seems this is a floating point underflow as the result of a fix to #18044. E.g. in pd.core.nanops.nankurt:

https://github.com/pandas-dev/pandas/blob/7945e563d36bcf4694ccc44698829a6221905839/pandas/core/nanops.py#L1354C2-L1359C47

```python3
    # floating point error
    #
    # #18044 in _libs/windows.pyx calc_kurt follow this behavior
    # to fix the fperr to treat denom <1e-14 as zero
    numerator = _zero_out_fperr(numerator)
    denominator = _zero_out_fperr(denominator)
```

Here the denominator is coming out as 2.4e-17 which is getting set to 0, because the numbers involved are so small.

Not sure how scipy is handling this to make it numerically stable for extremely small values.

**Comment by mroeschke** [comment]
Closing as a duplicate of https://github.com/pandas-dev/pandas/issues/57972
